server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -497,35 +542,34 @@ def parser_descriptivemetadata(self, item, descript_el):
         subjects += descript_el.findall('SubjectCode/Subject')
         item.setdefault('subject', []).extend(self.format_subjects(subjects))
 
-        # parser OfInterestTo element
-        elements = descript_el.findall('OfInterestTo')
-        if elements:
-            item['OfInterestTo'] = []
-            for element in elements:
-                item['OfInterestTo'].append(element.get('FormalName', ''))
+        # parser OfInterestTo is CV
+        for element in descript_el.findall('OfInterestTo'):
+            if element.get('FormalName'):
+                # genre CV
```

Comment:
this one is not `# genre CV` :)

Reply:
remove the comment, 
I have add values of "OfInterestTo" into vocabularies.json.

Reply:
We also add values 'OfInterestTo' into the vocabularies.json, is it right?